packages/integration-objects/srv/validations/ValidateFieldsForRateTypeDesc.js

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+const util = require('@sap-cloud-sdk/util');
+const { ValidationError } = require('../exceptions/validation-error');
+const ErrorStatuses = require('../utils/ErrorStatuses');
+const RateTypeExtensionConstants = require('../utils/RateTypeExtensionConstants');
+const { validateLength } = require('./ValidateLength');
+const exchangeRateTypeDescField = 'exchangeRateTypeDescription';
+const localeField = 'locale';
+const MAX_VALUE_LOCALE = 14;
+const MAX_VALUE_EXCHANGE_RATE_TYPE_DESCRIPTION = 30;
+
+async function validateFieldsForExchangeRateTypeDescription(data, params) {
```

Comment:
This function does not do anything `async`.

Reply:
Removed and updated